BaseFilter/sections/css_extended.txt

```diff
@@ -1908,6 +1908,7 @@ indiastudychannel.com#?##content_rightbar > div.roundedBlock:has(> div#HP-gpt-pa
 indiastudychannel.com#?##content_rightbar > div.roundedBlock:has(> div[id^="div-gpt-ad-"])
 !
 insta-stories-viewer.com#?#.context__caption > .context__title:contains(/^Advertising$/)
+sourceforge.net#?#.main-content > div.sterling > div[data-id^="div-gpt-ad-"]:upward(1)
```

Comment:
Seems this rule is enough. Please check it on your side and change it.
```adblock
sourceforge.net##.sterling
```

Reply:
I think so too. thank you.